clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/converters/DestroyTitusJobAtomicOperationConverter.groovy

```diff
@@ -58,6 +61,20 @@ class DestroyTitusJobAtomicOperationConverter extends AbstractAtomicOperationsCr
       converted.requiresApplicationRestriction = !converted.applications.isEmpty()
       converted.serverGroupName = job.name
     } catch (Exception e) {
+      if (e instanceof StatusRuntimeException) {
+        def statusRuntimeException = (StatusRuntimeException) e
+        if (statusRuntimeException.status.code == Status.NOT_FOUND.code) {
+          throw new NotFoundException(
```

Comment:
It seemed weird to just continue when we know up front that the job doesn't exist. That said, I have now made the downstream behavior handle a missing job.